.travis.yml

```diff
@@ -3,6 +3,9 @@ python:
   - "3.6"
   - "3.7"
   - "3.8"
+  - "3.9"
+  - "3.10"
+  - "3.11"
```

Comment:
Are there still projects around using Travis CI??

Reply:
I don't know. I just assumed it was still using it. I wouldn't mind switching it to GitHub CI.

Reply:
I can write something up pretty quickly for GitHub CI in another PR if that interests you.

Reply:
I can copy-paste a config from another repository as well. 
It is not a too complicated CI definition.

Reply:
I'll pass you something; I've been doing this all week, haha.

Reply:
Already done 😅